java/README.md

```diff
@@ -3,19 +3,19 @@
 This example will show you how to build your Java IronWorker, test it locally, then upload it
 to IronWorker for production.
 
-**Note**: Be sure you've followed the base [getting started instructions on the top level README](https://github.com/iron-io/dockerworker). 
+**Note**: Be sure you've followed the base [getting started instructions on the top level README](https://github.com/iron-io/dockerworker).
 
 First, let's build it on the right architecture using the actual Docker image it will be running on. The
-dependencies are passed into javac. 
+dependencies are passed into javac.
 
 ```sh
-docker run --rm -v "$(pwd)":/worker -w /worker iron/images:java-1.8 sh -c 'javac -cp "json-java.jar:gson-2.2.4.jar:ironworker.jar" Worker101.java PayloadData.java'
+docker run --rm -v "$(pwd)":/worker -w /worker iron/java-dev:1.8 sh -c 'javac -cp "json-java.jar:gson-2.2.4.jar:ironworker.jar" Worker101.java PayloadData.java'
 ```
 
 Now run it to test it out:
 
 ```sh
-docker run --rm -v "$(pwd)":/worker -w /worker iron/images:java-1.8 sh -c 'java -cp gson-2.2.4.jar:json-java.jar:ironworker.jar:. Worker101 -payload hello.payload.json -config hello.config.yml -id 123'
+docker run --rm -v "$(pwd)":/worker -w /worker iron/java:1.8 sh -c 'java -cp gson-2.2.4.jar:json-java.jar:ironworker.jar:. Worker101 -payload hello.payload.json -config hello.config.yml -id 123'
```

Comment:
@treeder it's like this in `master` now. If I switch to env vars, the code throws a null pointer exception. Looks like the problem is on [line 24 of Worker101.java](https://github.com/iron-io/dockerworker/blob/master/java/Worker101.java#L24) since we pass cmdline args to `WorkerHelper.fromArgs`. Unless it's an easy fix or I have this all wrong, I'd rather save the code change for a later PR.